crates/sui-graphql-rpc/src/lib.rs

```diff
@@ -16,4 +16,5 @@ mod mutation;
 pub(crate) mod raw_query;
 pub mod server;
 pub mod test_infra;
+pub(crate) mod tx_lookups;
```

Comment:
(nit, I realise this is a draft so this will probably change soon), this is kind of a random place for this module -- I reckon it belongs as a sub-module of `types::transaction_block` or something.

Reply:
Ah interestingly this was in transaction block first but I took it out into its own file lol

Reply:
Gotcha, if so that makes sense, happy to set things up accordingly